### PR TITLE
enable tiff

### PR DESCRIPTION
**Pull request by nicolabeghin**

### CODE REVIEW

**Comment by dooman87** [comment]
Thanks a lot for the PR! Would you mind adding a test please? That would require to add `test.tiff` and the command into `.travis.yml`

**Comment by nicolabeghin** [comment]
sure @dooman87 ! will do asap, but first I need to align the other Dockerfile.* since I updated only the `Docker.buster` one.

File: Dockerfile.buster [hunk 1]
```diff
@@ -31,7 +33,7 @@ RUN apt-get -y update && \
     # Building ImageMagick
     git clone https://github.com/ImageMagick/ImageMagick.git && \
     cd ImageMagick && git checkout ${IM_VERSION} && \
-    ./configure --without-magick-plus-plus --disable-docs --disable-static && \
+    ./configure --without-magick-plus-plus --disable-docs --disable-static --with-libtiff && \
     make && make install && \
     ldconfig /usr/local/lib && \
     apt-get remove --autoremove --purge -y gcc make cmake curl g++ yasm git autoconf pkg-config libpng-dev libjpeg62-turbo-dev libwebp-dev libde265-dev libx265-dev && \
```
Comment (dooman87) [inline Dockerfile.buster:39]: We could probably delete `libtiff-dev` here to keep image size as small as possible

Comment (nicolabeghin) [reply Dockerfile.buster:39]: Unfortunately that's exactly what made the trick - without installing `libtiff-dev` imagemagick didn't show tiff support even if -`-with-libtiff` was enabled!

Comment (dooman87) [reply Dockerfile.buster:39]: Interesting, I thought that you would need *-dev dependency during build time, but not to run.

Comment (nicolabeghin) [reply Dockerfile.buster:39]: I think I misunderstood your point, we _can_ remove _after_ compilation - will do!

**Comment by dooman87** [comment]
I cherry picked TIFF support from this PR, so we can close it. Thanks @nicolabeghin for the contribution!

**Pull request closed without merging**